profiler/src/main/java/com/splunk/opentelemetry/profiler/LogsExporterBuilder.java

```diff
@@ -47,10 +46,8 @@ static LogsExporter fromConfig(Config config) {
     return builder.build();
   }
 
-  private static ResourceLogsAdapter buildResourceLogsAdapter(Config config) {
-    Resource resource =
-        OpenTelemetryResourceAutoConfiguration.configureResource(
-            new ConfigPropertiesAdapter(config));
+  private static ResourceLogsAdapter buildResourceLogsAdapter() {
+    Resource resource = ResourceHolder.getResource();
```

Comment:
It's a bummer to have this static call here that's dependent on external prior dependencies. Can we pass the resource in and have the `JfrActivator` do the static work (and also override `order()` to do the right thing)?

Reply:
I wasn't aware of the issue. I just mimicked what Anuraag did to  x-ray sampler in otel contrib. In `JfrActivator` you won't need to worry about `order()` as it runs in `afterAgent`.

Reply:
moved static call to `JfrActivator`